EIPS/eip-6963.md

```diff
@@ -135,101 +91,83 @@ interface EIP6963ProviderDetail {
 }
 ```
 
-The `EIP1193Provider` interface is documented at [EIP-1193](./eip-1193.md) and can be used to override the `window.ethereum` object once the user has explicitly selected it.
-
 ### Window Event
 
-Different wallet providers might inject their scripts at different times, plus there is no guarantees that the Ethereum library in the web page will be loaded before all injected scripts are present in the page.
+In order to prevent provider collisions, the DApp and the Wallet are expected to emit an event and instantiate an eventListener to discover the various Wallet's. This forms an Event concurrency loop. 
```

Comment:
While "Wallet" is used as a term, I don't think an apostrophe is needed for pluralizing it